floor/floor/server/static/playlist-controls.js

```diff
@@ -146,6 +146,23 @@ function handleTempoUpdate(tempoData) {
     status.value = tempoData.bpm;
 }
 
+function setBrightness(brightness) {
+    var data = {
+        brightness: brightness,
+    };
+    console.log('Setting brightness to: ', brightness);
+    return axios.post('/api/brightness', data).then(function (response) {
+        handleBrightnessUpdate(response.data);
+    }).catch(function (error) {
+        handleError(error);
+    });
+}
+
+function handleBrightnessUpdate(brightnessData) {
+    var brightness = document.getElementById('brightness');
+    brightness.value = brightnessData.brightness;
```

Comment:
Is it necessary to update the dropdown menu here since the user has just changed it to the value they want?  If yes, then we should make sure `brightnessData.brightness` returns `1.0`.  Right now its returning `1`, so when you select "100%" from the interface, it updates the server, then the response sets the dropdown to a non-existent option and blanks it out.

Reply:
I could see us having the periodic status call update brightness as well so that it matches any updates coming from another source (e.g. the midi controller)

Reply:
Added this to `getStatus` - it will be continuously sync'd along with the others stuff :+1: